src/pages/ReimbursementAccount/PersonalInfo/PersonalInfo.js

```diff
@@ -0,0 +1,96 @@
+    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
+};
+
+const STEPS_HEADER_HEIGHT = 40;
+
+function PersonalInfo({reimbursementAccount}) {
+    const {translate} = useLocalize();
+
+    const submit = useCallback(() => {
+        const values = {
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.FIRST_NAME], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.LAST_NAME], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.DOB]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.DOB], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.SSN_LAST_4]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.SSN_LAST_4], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.STREET]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.STREET], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.CITY]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.CITY], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.STATE]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.STATE], ''),
+            [CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.ZIP_CODE]: lodashGet(reimbursementAccount, ['achData', CONST.BANK_ACCOUNT.PERSONAL_INFO_STEP.INPUT_KEY.ZIP_CODE], ''),
+        };
+
+        const payload = {
+            bankAccountID: _.get(reimbursementAccount, 'achData.bankAccountID', 0),
+            ...values,
+        };
+
+        BankAccounts.updatePersonalInformationForBankAccount(payload);
+    }, [reimbursementAccount]);
+
+    const startFrom = useMemo(() => getInitialSubstepForPersonalInfo(lodashGet(reimbursementAccount, ['achData'], {})), [reimbursementAccount]);
+    const bodyContent = [FullName, DateOfBirth, SocialSecurityNumber, Address, Confirmation];
+
+    const {componentToRender: SubStep, isEditing, screenIndex, nextScreen, prevScreen, moveTo} = useSubStep({bodyContent, startFrom, onFinished: submit});
+
+    const handleBackButtonPress = () => {
+        if (screenIndex === 0) {
+            Navigation.goBack(ROUTES.HOME);
+        } else {
+            prevScreen();
+        }
+    };
+
+    return (
+        <>
+            <HeaderWithBackButton
+                onBackButtonPress={handleBackButtonPress}
+                title={translate('personalInfoStep.personalInfo')}
+            />
+            <View style={[styles.ph5, styles.mv3, {height: STEPS_HEADER_HEIGHT}]}>
+                <InteractiveStepSubHeader
+                    onStepSelected={() => {}}
+                    startStep={1}
```

Comment:
I believe it's the second step but more importantly we should probably get it from onyx. I know it's mocked for now but do we have this thought out?

Reply:
count starts from 0 :D

Reply:
ofc it does xD got confused with the naming of steps below 😅